app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,23 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def validate_authorize_form
+      result = @authorize_form.check_submit
+      return if result.success?
+
+      track_authorize_analytics(result)
```

Comment:
Should this line come before the `return` on line 76? Otherwise, it will only track the failure scenario.